extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeMixPlaylistExtractorTest.java

```diff
@@ -41,6 +43,7 @@
     private static final String VIDEO_ID = "_AzeUSL9lZc";
     private static final String VIDEO_TITLE =
             "Most Beautiful And Emotional  Piano: Anime Music Shigatsu wa Kimi no Uso OST IMO";
+    private static final String RESOURCE_PATH = "src/test/resources/org/schabi/newpipe/extractor/services/youtube/mix/";
```

Comment:
The base resource path should be put in some central class, like in `DownloaderFactory`. The root path should in that case be `"src/test/resources/org/schabi/newpipe/extractor/"`

Reply:
done